xbmc/input/actions/ActionIDs.h

```diff
@@ -53,6 +53,7 @@
 #define ACTION_TOGGLE_SOURCE_DEST     32 //!< used to toggle between source view and destination view. Can be used in myfiles.xml window id=3
 #define ACTION_SHOW_PLAYLIST          33 //!< used to toggle between current view and playlist view. Can b used in all mymusic xml files
 #define ACTION_QUEUE_ITEM             34 //!< used to queue a item to the playlist. Can b used in all mymusic xml files
+#define ACTION_PLAY_NEXT_ITEM        259 //!< used to queue an item to the next position in the playlist
```

Comment:
This is sorted by the number, so move to the correct place. Otherwise one may think it's unused later.

Reply:
It's kinda not sorted (see line 47 which is why I wasn't sure about it) but I'll move it anyways :)
Also gonna give it `247` as I noticed that one being free as well

Reply:
lol wow.
247 is exactly the idea which is out of line and I re-used it.